application/src/main/java/org/togetherjava/tjbot/commands/CommandHandler.java

```diff
@@ -51,10 +55,29 @@ public void onReady(@NotNull ReadyEvent event) {
                             new CommandData(command.getCommandName(), command.getDescription())))
                         .queue();
                 }
+            }, throwable -> {
+                handleErrors(throwable, guild);
             });
         });
     }
 
+    private static void handleErrors(Throwable throwable, Guild guild) {
+        new ErrorHandler().handle(ErrorResponse.MISSING_ACCESS, errorResponseException -> {
+            // * Horrible, but there's no other way really
```

Comment:
`// Note Horrible, ...`. Also, could you mention what is horrible about it?

Reply:
Looping through all channels of a guild to find a channel to set a message in.
There's no other way, but causes it to be extremely unreadable imo